pushy/src/main/java/com/turo/pushy/apns/proxy/HttpProxyHandlerFactory.java

```diff
@@ -89,4 +142,16 @@ public ProxyHandler createProxyHandler() {
 
         return handler;
     }
+
+    /* (non-Javadoc)
+     * @see java.lang.Object#toString()
+     */
+    @Override
+    public String toString() {
```

Comment:
I'm a little hesitant about adding this `toString` method, especially as implemented. Can we either drop this (if it was only intended for development purposes) or, if we think there's an ongoing need, avoid including `password` in cleartext?

Reply:
That's a major duh on a cleartext password, sorry.

I found it useful when deploying in order to show that my settings were making their way through to the client. I'll put it in with a fixed number of asterisks if OK?